actors/test/actor-tests/src/main/java/cloud/orbit/actors/test/ActorBaseTest.java

```diff
@@ -486,6 +486,24 @@ protected void eventuallyTrue(long timeoutMillis, final Callable<Boolean> callab
 
     }
 
+    protected void waitForDeactivations() {
+        boolean isBusy;
+        do
+        {
+            isBusy = false;
+            for(Stage stage : stages)
+            {
+                System.out.println(stage);
+                if(stage.getLocalObjectsCleaner().hasBacklog())
+                {
+                    isBusy = true;
+                    break;
+                }
+
+            }
+        } while(isBusy);
```

Comment:
Yes, it's just in tests though. Cleanup now drops straight through so you can't expect any deactivations to actually have taken place.